### PR TITLE
Fixes #6304: Added eye icon for contributors that are following too

### LINKED ISSUE #6304: Reorganize /contributors/___ pages
(follow-up to https://github.com/publiclab/plots2/pull/5632) cc @IshaGupta18 🙌 

We now show 2 types of users on our contributor pages like https://publiclab.org/contributors/water-quality

1. contributors - who have posted using this topic or commented on this topic
2. followers - who are following this topic and will get notified when new content is posted to it

They're a bit messily next to each other:

image

Let's:

* [ ] show contributors in a table on top, but in each row, mark if that person is ALSO following -- maybe, with an eye icon? In a column called `Following`, maybe?)
* [ ] then let's show people who are NOT in the first list, but ARE subscribed. This can be in a second table with the title `Following`. 

Should we have a short text explanation of the distinction?

### PR DESCRIPTION
**Pull request by KarishmaVanwari**
An eye icon now appears right beside the name of the contributor if it is following too.

Fixes #6304 

Screeenshot:

image

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/tag/_contributors.html.erb [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
+            <tr class="followers-row">
+              <td><a href='/profile/<%= follower.name %>'><i class="fa fa-user-o"></i> <%= follower.name %></a></td>
+            </tr>
+          <% end %>
+        </table>
+      <% end %>
+    </div>
+  </div>
+  <br />
+<% end %>
```
Comment (TildaDares) [inline app/views/tag/_contributors.html.erb:76]: Hi @KarishmaVanwari, I think your linter/formatter added more changes than necessary.

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by TildaDares** [comment]
Hi @KarishmaVanwari, the changes are still there.

File: app/views/tag/_contributors.html.erb (hunk 1 above)
Comment (TildaDares) [reply app/views/tag/_contributors.html.erb]: Hi @KarishmaVanwari, the changes are still there.

Comment (KarishmaVanwari) [reply app/views/tag/_contributors.html.erb]: Yes @TildaDares. I'll close this PR and create another one. Apologies for the inconvenience.

**Comment by KarishmaVanwari** [comment]
@TildaDares Closing this PR since I've created another PR #10794

**Pull request closed without merging**